formatTest/typeCheckedTests/input/jsx.re

```diff
@@ -332,8 +332,8 @@ let myFun () => {
 /* </Foo>; */
 
 /* let res = <Foo a=10 b=(<Foo a=200 />) />; */
-let zzz = "oh hai";
-let optionalCallSite = <Optional1 optional=?zzz />;
+let zzz = Some "oh hai";
+let optionalCallSite = <Optional1 requiredOpt=?zzz />;
 fakeRender optionalCallSite;
 let optionalArgument = <Optional2 />;
```

Comment:
Also for `DefaultArg` below as well. To make sure that it's parsed correctly - we should force passing an `option` datatype via `<DefaultArg optional=?someIdentifier />`